vault/resource_pki_secret_backend_role.go

```diff
@@ -196,6 +197,12 @@ func pkiSecretBackendRoleResource() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			consts.FieldAllowedURISansTemplate: {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
```

Comment:
The default value from the API is "false" so we are safe to set this boolean as "Computed"